tools/bazelwrapper

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+# bazelwrapper
+# A script to sit between your scripts and bazel and can be checked into your
+# repository. Bazel moves quickly and is not always backwards compatible so the
+# version of bazel each commit builds with is important to preserve history.
+# This script can update bazel version by modifying the BAZEL_VERSION,
+# BAZEL_VERSION_SHA below to the desired release
+# (https://github.com/bazelbuild/bazel/releases)
+
+set -e
+
+# Make sure we're in the project root directory.
+SCRIPTPATH="$( cd "$( dirname "${BASH_SOURCE[0]}"  )" && pwd  )"
+pushd "$SCRIPTPATH/.." > /dev/null
+trap popd > /dev/null ERR EXIT
+
+# Go to bazel release page
+# These are typically posted in groups.google
+# https://groups.google.com/forum/#!forum/bazel-discuss
+BAZEL_VERSION="0.19.2"
+BAZEL_VERSION_SHA="25ea85d4974ead87a7600e17b733bf8035a075fc8671c97e1c1f7dc8ff304231"
+BAZEL_VERSION_URL="https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-installer-darwin-x86_64.sh"
+
+BAZEL_ROOT="$HOME/.bazelenv/versions/$BAZEL_VERSION"
+BAZEL_PATH="$BAZEL_ROOT/bin/bazel"
+XCODE_SELECT_ENV_PATH="${SCRIPTPATH}/.xcode_select_env"
+LEGACY_BAZEL_PATH="$SCRIPTPATH/../Scripts/bazel/bin/bazel"
+
+BAZEL=""
+
+function install_bazel() {
+    curl -L "$BAZEL_VERSION_URL" > $PWD/install_bazel.sh
+    SHA=$(shasum -a 256 install_bazel.sh | awk '{ print $1 }')
+    if [[ $SHA == $BAZEL_VERSION_SHA ]]; then
+        chmod +x install_bazel.sh
+        $PWD/install_bazel.sh --prefix="$BAZEL_ROOT" && rm install_bazel.sh
+    else
+        echo "You version of bazel is out of date; ask for help in #cx-ios"
+        exit 1
+    fi
+}
+
+# Check if we have the correct version of bazel installed in the home
```

Comment:
We can remove this, it's sort of a Pinterest legacy codebase comment and isn't applicable here